api/v1alpha4/types.go

```diff
@@ -116,6 +116,32 @@ type SubnetFilter struct {
 	NotTagsAny      string `json:"notTagsAny,omitempty"`
 }
 
+type PortOpts struct {
+	// ID of the OpenStack network on which to create the port. If unspecified, create the port on the default cluster network.
+	NetworkID string `json:"networkId,omitempty"`
+	// Used to make the name of the port unique. If unspecified, instead the 0-based index of the port in the list is used.
+	NameSuffix   string `json:"nameSuffix,omitempty"`
+	Description  string `json:"description,omitempty"`
+	AdminStateUp *bool  `json:"adminStateUp,omitempty"`
+	MACAddress   string `json:"macAddress,omitempty"`
+	// Specify pairs of subnet and/or IP address. These should be subnets of the network with the given NetworkID.
+	FixedIPs       []FixedIP `json:"fixedIps,omitempty"`
```

Comment:
```suggestion
	FixedIPs       []FixedIP `json:"fixedIPs,omitempty"`
```

Reply:
I now JSON capitalization is a mess in CAPO. I think I would capitalize IP for now and probably fix all the `Id`s in a follow-up issue. (at least we're still in alpha so we can fix things like this :))